.github/workflows/update-layers.yaml

```diff
@@ -23,7 +26,7 @@ jobs:
           role-duration-seconds: 1800
       - name: Run Copy Layer
         run: |
-          ./copy-layer.sh
+          lambdalayer/copy-layer.sh
```

Comment:
I think this still wants to be `./lambdalayer/copy-layer.sh` 🙈 
Or `bash lambdalayer/copy-layer.sh` would work just as fine.